src/uct/ib/base/ib_md.h

```diff
@@ -64,6 +64,10 @@ enum {
     UCT_IB_DEVX_OBJ_DCI
 };
 
+enum {
+    UCT_IB_MD_FLAG_DC = UCS_BIT(0)
```

Comment:
1. IMO we should not have any mention of DC in the base classes of verbs
2. what about disabling rc_mlx5?

Reply:
> * what about disabling rc_mlx5?

do you mean disable mlx5 ifaces at all? UD too?

Reply:
sure
mlx5 transport should run only on mlx5-enabled md
right @dmitrygx ?

Reply:
> mlx5 transport should run only on mlx5-enabled md
> right @dmitrygx ?

right

Reply:
maybe we should have some `UCT_IB_MD_FLAG_MLX5` flag which indicate support of MLX5-capable devices?

Reply:
fixed